### PR TITLE
Add GPT fdisk 1.0.1

### PR DESCRIPTION
**Pull request by alarcher**

### CODE REVIEW

File: components/sysutils/gptfdisk/gptfdisk.p5m [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.summary value="$(COMPONENT_SUMMARY)"
+set name=info.classification value="$(COMPONENT_CLASSIFICATION)"
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
+
```
Comment (pyhalov) [inline components/sysutils/gptfdisk/gptfdisk.p5m:24]: Do we want isaexec wrappers?

Comment (alarcher) [inline components/sysutils/gptfdisk/gptfdisk.p5m:24]: I can add them, no problem.

Comment (alarcher) [inline components/sysutils/gptfdisk/gptfdisk.p5m:24]: Done.

File: components/sysutils/gptfdisk/files/Makefile.solaris [hunk 2]
```diff
@@ -0,0 +1,57 @@
+CC=gcc
```
Comment (pyhalov) [inline components/sysutils/gptfdisk/files/Makefile.solaris]: Spote one minor issue. Would be better to use CC ?= ... and pass CC and CXX variables in the environment

Comment (alarcher) [inline components/sysutils/gptfdisk/files/Makefile.solaris]: OK if you like.

Comment (alarcher) [inline components/sysutils/gptfdisk/files/Makefile.solaris]: Done.

**Pull request merged**